### ISSUE TITLE
In file with multiple enums only first enum processed properly

### ISSUE THREAD

**Issue by DronCode** [DronCode/RG3#6]
rg3py version: `rg3py==0.0.16`

Sample code:

```
namespace engine::render
{
	/**
	 * @runtime
	 * @serializer
	 */
	enum class EBlendOperand
	{
		NO_OPERAND = 0,

		BLEND_OP_ZERO,
		BLEND_OP_ONE,
		BLEND_OP_SRC_COLOR,
		BLEND_OP_ONE_MINUS_SRC_COLOR,
		BLEND_OP_DEST_COLOR,
		BLEND_OP_ONE_MINUS_DEST_COLOR,
		BLEND_OP_SRC_ALPHA,
		BLEND_OP_ONE_MINUS_SRC_ALPHA,
		BLEND_OP_CONST_COLOR,
		BLEND_OP_ONE_MINUS_CONST_COLOR,
		BLEND_OP_CONST_ALPHA,
		BLEND_OP_ONE_MINUS_CONST_ALPHA
	};
}

// Workaround for RG3
namespace engine::render
{
	/**
	 * @runtime
	 * @serializer
	 */
	enum class EBlendFunction
	{
		NO_FUNCTION = 0,

		BLEND_ADD,
		BLEND_SUBTRACT,
		BLEND_REVERSE_SUBTRACT,
		BLEND_MIN,
		BLEND_MAX
	};
}
```

second type not processed properly.

**Comment by DronCode** [comment]
Fixed in commit https://github.com/DronCode/RG3/commit/bd77781edd17f44f499812bdbb47218d94eff949
Will be in relese 0.0.17

**Comment by DronCode** [comment]
Fixed in 0.0.17
Done
